### ISSUE TITLE
Fail to complete the activity of undeleting blobs 

### ISSUE THREAD

**Issue by v-xuanzh** [Microsoft/AzureStorageExplorer#642]
**Storage Explorer Version**: 1.5.0 
**Platform/OS Version**: Windows 10/ Linux Ubuntu 16.04/ MacOS High Sierra
**Architecture**: ia32
**Build Number**:20180905.3
**Commit**: 66ac14b9
**Regression From**: Previous release 1.4.1(20180822.1)

#### Steps to Reproduce: ####
1. Open one blob container with soft delete enabled.
2. Upload a blob to it -> Try to delete the uploaded blob -> Switch to 'Active and deleted blobs'.
3. Right click the deleted blob then select 'Undelete Selected'.

#### Expected Experience: ####
The deleted blob can be undeleted successfully.

#### Actual Experience: ####
Fail to complete the activity.
![loading](https://user-images.githubusercontent.com/34729022/45208054-34b71200-b279-11e8-8c0e-451a2c469a1a.png)

**Comment by craxal** [comment]
OK, so the activity never completes. Does the blob itself reappear if you refresh the blob list?

**Comment by v-xuanzh** [comment]
No, the blob still displays as deleted after refresh the blob list.

**Comment by v-xianya** [comment]
Verified on build 20190108.2. This issue doesn't reproduce. Could you help to resolve it?
